mycore-acl/src/main/java/org/mycore/mcr/acl/accesskey/restapi/v2/MCRRestObjectAccessKeys.java

```diff
@@ -67,18 +76,18 @@ public class MCRRestObjectAccessKeys {
     @Operation(
         summary = "Lists all access keys for an object",
         responses = {
-            @ApiResponse(responseCode = "200",
+            @ApiResponse(responseCode = OK,
                 description = "List of access keys attached to this metadata object",
                 content = { @Content(mediaType = MediaType.APPLICATION_JSON,
                     array = @ArraySchema(schema = @Schema(implementation = MCRAccessKey.class))) }),
-            @ApiResponse(responseCode = "" + MCRObjectIDParamConverterProvider.CODE_INVALID, // 400
+            @ApiResponse(responseCode = BAD_REQUEST,
                 description = MCRObjectIDParamConverterProvider.MSG_INVALID,
                 content = { @Content(mediaType = MediaType.APPLICATION_JSON) }),
-            @ApiResponse(responseCode = "401",
-                description = "You do not have create permission and need to authenticate first",
+            @ApiResponse(responseCode = UNAUTHORIZED,
+                description = MCRRestDescription.UNAUTHORIZED,
                 content = { @Content(mediaType = MediaType.APPLICATION_JSON) }),
-            @ApiResponse(responseCode = "404",
-                description = "Object or access key does not exist",
+            @ApiResponse(responseCode = NOT_FOUND,
+                description = MCRRestDescription.OBJECT_NOT_FOUND,
```

Comment:
`MCRRestStatusCode.NOT_FOUND` is not equivalent to _Object or access key does not exist_
Create and use local member.